components/datetime/src/scaffold/fieldset_traits.rs

```diff
@@ -37,8 +37,8 @@ pub trait DateInputMarkers: UnstableSealed {
     type DayOfYearInput: IntoOption<DayOfYear>;
     /// Marker for resolving the day-of-year input field.
     type RataDieInput: IntoOption<RataDie>;
-    /// Marker for resolving the day-of-week input field.
-    type DayOfWeekInput: IntoOption<Weekday>;
+    /// Marker for resolving the weekday input field.
+    type WeekdayInput: IntoOption<Weekday>;
```

Comment:
See comment on the other PR about semver question

Reply:
> /// 🚧 This trait is considered unstable; it may change at any time, in breaking or non-breaking ways,